src/current/_includes/molt/molt-troubleshooting-replication.md

```diff
@@ -0,0 +1,257 @@
+### Forward replication issues
+
+##### Performance troubleshooting
+
+If MOLT Replicator appears hung or performs poorly:
+
+1. Enable trace logging with `-vv` to get more visibility into the replicator's state and behavior.
+
+1. If MOLT Replicator is in an unknown, hung, or erroneous state, collect performance profiles to include with support tickets:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    curl 'localhost:30005/debug/pprof/trace?seconds=15' > trace.out
```

Comment:
I wonder if it's better here to show the template for how to do this? i.e. `<your host>:<your metrics port>/debug/pprof/trace`. Maybe we can keep this code block as a concrete example, but before it, mention that the URL should be in the format mentioned in the sentence before?